tools/stress-cluster/cluster/kubernetes/stress-infrastructure/Chart.yaml

```diff
@@ -29,5 +29,5 @@ dependencies:
   repository: https://charts.chaos-mesh.org
   condition: deploy.chaosmesh
 - name: stress-test-addons
-  version: 0.1.12
-  repository: https://stresstestcharts.blob.core.windows.net/helm/
+  version: 0.1.16
+  repository: "@stresstestcharts"
```

Comment:
Will this break if we ran provision on this?

Reply:
Yes, in lieu of your updates to the provision script to change the script to add `helm repo add` [here](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/provision.ps1#L153). Looking through that code, I'm realizing that there's a bug where it never adds the stresstestcharts repo regardless. We just luck out because the deploy script already installs it for our machine [here](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1#L97).

Reply:
Going to revert that change to keep things in a working state, since your PR is forthcoming and we have examples in the readme.